bot.py

```
@@ -1,11 +1,12 @@
 # This example requires the 'message_content' intent.
 
 import discord
-from discord.ext import tasks
+from discord.ext import commands
 import asyncio
 import re
 import os
 import json
+from typing import Callable, Optional
```

Ruff (0.11.9)

`typing.Callable` imported but unused

Remove unused import: `typing.Callable`

(F401)